website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -0,0 +1,44 @@
+---
+page_title: Destruction and Deletion - Workspaces - Terraform Cloud
+description: |-
+  Learn about destroying infrastructure and deleting workspaces in Terraform Cloud.
+---
+
+# Destruction and Deletion
+
+Terraform Cloud workspaces have two primary delete actions:
+
+- [Destroying infrastructure](#destroy-infrastructure) deletes resources managed by the Terraform Cloud workspace by triggering a destroy run.
+- [Deleting a workspace](#delete-workspaces) deletes the workspace itself without triggering a destroy run.
+
+In general, you should perform both actions in the above order when destroying a workspace to ensure resource cleanup for all of a workspace's managed infrastructure.
+
+## Destroy Infrastructure
+
+Destroy plans delete the infrastructure managed by a workspace. We recommend destroying the infrastructure managed by a workspace _before_ deleting the workspace itself. Otherwise, the unmanaged infrastructure resources will continue to exist but will become unmanaged, and you must got into your infrastructure providers to delete the resources manually.
```

Comment:
nit: typo here? "you must got into your infrastructure providers"

Reply:
I mean, this is a docs PR... so would typos really be a nit? 😅 Fixed!